packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CalendarDate,
+    getLocalTimeZone,
+    getWeeksInMonth,
+    isSameDay,
+    startOfMonth,
+    startOfWeek,
+    today,
+} from '@internationalized/date';
+import {
+    CSSResultArray,
+    html,
+    PropertyValueMap,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import { LanguageResolutionController } from '@spectrum-web-components/reactive-controllers/src/LanguageResolution.js';
+
+import { property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { when } from 'lit/directives/when.js';
+
+import { CalendarWeekday, daysInWeek } from './types.js';
+import styles from './calendar.css.js';
+
+import '@spectrum-web-components/action-button/sp-action-button.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-left.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-right.js';
+
+/**
+ * @element sp-calendar
+ *
+ * @slot prev-icon - The icon used in the _"Previous Month"_ button
+ * @slot next-icon - The icon used in the _"Next Month"_ button
+ *
+ * @event change - Announces when a day is selected by emitting a `Date` object
+ */
+export class Calendar extends SpectrumElement {
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    padded = false;
+
+    @property({ type: Boolean, reflect: true })
+    disabled = false;
+
+    @property({ reflect: true, attribute: false })
+    selectedDate?: Date;
+
+    @property({ reflect: true, attribute: false })
+    min?: Date;
+
+    @property({ reflect: true, attribute: false })
+    max?: Date;
```

Comment:
If you use JSDoc comments on these properties, they will appear in the API docs for this components: https://paulodetarsofm-calendar--spectrum-web-components.netlify.app/components/calendar/api/